test/second_q/problems/test_vibrational_properties_container.py

```diff
@@ -35,7 +32,7 @@ def test_occupied_modals(self) -> None:
 
         with self.subTest("wrong setting type"):
             with self.assertRaises(TypeError):
-                container.occupied_modals = ElectronicDipoleMoment()  # type: ignore[assignment]
+                container.occupied_modals = Magnetization(2)  # type: ignore[assignment]
```

Comment:
I'm just using some other non-vibrational property, since the `ElectronicDipoleMoment` no longer has an empty-init available